### PR TITLE
Add devcontainer for Bash

### PR DESCRIPTION
**Pull request by AarynSmith**
Adds a configuration for Bash development for #254. Included Bash IDE and Bash Debug extensions.

### CODE REVIEW

File: containers/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Summary
+
+*Develop scripts with Bash, includes [Bash IDE](https://marketplace.visualstudio.com/items?itemName=mads-hartmann.bash-ide-vscode), and [Bash Debug](https://github.com/rogalmic/vscode-bash-debug).*
+
+|          Metadata           |                    Value                     |
+| --------------------------- | -------------------------------------------- |
+| *Contributors*              | [Aaryn Smith](https://gitlab.com/aarynsmith) |
+| *Definition type*           | Dockerfile                                   |
+| *Works in Codespaces*       | Yes                                          |
```
Comment (Chuxel) [inline containers/bash/README.md:11]: Similar comment on Codespaces here as Hugo

File: containers/bash/.devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM alpine:latest
```
Comment (Chuxel) [inline containers/bash/.devcontainer/Dockerfile:1]: Also same as in the other case, you could use the pre-built `mcr.microsoft.com/vscode/devcontainers/base:alpine` image if you want. I think all the dependencies in the Dockerfile are covered, so you'd be set.

File: containers/bash/README.md (hunk 1 above)
Comment (AarynSmith) [reply containers/bash/README.md:11]: Updated

File: containers/bash/.devcontainer/Dockerfile (hunk 2 above)
Comment (AarynSmith) [reply containers/bash/.devcontainer/Dockerfile:1]: Updated.

**Comment by Chuxel** [comment]
LGTM!

**Pull request merged**